docs/04.Cloud-Native/4.1.Kubernetes-Ingress-Controller.md

```diff
@@ -201,3 +202,17 @@ httpServer:
   keepAliveTimeout: 60s
   maxConnections: 10240
 ```
+
+### Additional Annotations
+
+When defining your `Ingress` configurations, you can use the following `easegress` specific annotations for added customization:
+
+- **Load Balancing**: `easegress.ingress.kubernetes.io/proxy-load-balance`
+- **Header Hash Key for Load Balancing**: `easegress.ingress.kubernetes.io/proxy-header-hash-key`
+- **Forwarding Key**: `easegress.ingress.kubernetes.io/proxy-forward-key`
+- **Server Response Maximum Size**: `easegress.ingress.kubernetes.io/proxy-server-max-size`
+- **Proxy Timeout**: `easegress.ingress.kubernetes.io/proxy-timeout`
```

Comment:
Could you please add some examples here? 
Like [nginx](https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/). So users can understand it better or simply copy it to their yamls.

Reply:
Updated